script.js

```diff
@@ -0,0 +1,39 @@
+// global variables 
+    // added by Mila
+
+
+    //added by John
+
+
+    //added by Kephane
+
+
+
+
```

Comment:
That is a very good initiative, Mila. We could bring this topic up during our presentation, on Saturday! 
something like:
''One of the obstacles we had to deal with was coordinating all of our variable's names, and making sure there weren't any duplicates, to avoid unexpected results after a merge. ...''

Looking good!

Reply:
Good point！